src/xhity.c

```diff
@@ -13977,6 +13977,11 @@ int vis;						/* True if action is at all visible to the player */
 			elemdmg += 6 + weapon->spe;
 			totldmg += 6 + weapon->spe;
 		}
+	} else if(weapon && weapon->oartifact == ART_TOBIUME && !Shock_res(mdef)){
```

Comment:
This `else if` could just be an `||` in the original `if`, since the bodies of the two are identical.
`if(weapon && weapon->oartifact == ART_TOBIUME && !(Fire_res(mdef) || Shock_res(mdef))`